src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4375,6 +4375,73 @@ class MavlinkStreamCameraCapture : public MavlinkStream
 	}
 };
 
+class MavlinkStreamGpsGlobalOrigin : public MavlinkStream
```

Comment:
Can you add this under the `streams/` folder?

Reply:
You mean as a header like `streams/RAW_RPM.hpp`? Sure.

Reply:
Done and tested, force push ✔️